conf/config.json

```diff
@@ -21,6 +21,9 @@
         }
     },
     "extensions": {
+        "lifecycle": {
+            "zookeeperPath": "/lifecycle/data/buckets"
```

Comment:
This variable should reflect the common base path for all lifecycle matters, so `/lifecycle` only.

I'm going to open a PR with the basic configuration shared by all lifecycle components.